dev-libs/hidapi/hidapi-0.9.0.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Library for Interfacing with HID-Class devices"
+DESCRIPTION="HIDAPI is a multi-platform library which allows an application to \
+interface with USB and Bluetooth HID-Class devices on Windows, Linux, and \
+macOS. While it can be used to communicate with standard HID devices like \
+keyboards, mice, and Joysticks, it is most useful when used with custom \
+(Vendor-Defined) HID devices. Many devices do this in order to not require a \
+custom driver to be written for each platform. HIDAPI is easy to integrate with \
+the client application, just requiring a single source file to be dropped into \
+the application. (On Haiku, it only works with USB)."
+HOMEPAGE="http://www.signal11.us/oss/hidapi/"
+COPYRIGHT="2010 Alan Ott
+	2011 Signal 11 Software"
+LICENSE="BSD (3-clause)
+	GNU GPL v3"
+REVISION="2"
+SOURCE_URI="https://github.com/libusb/hidapi/archive/hidapi-$portVersion.tar.gz"
+CHECKSUM_SHA256="630ee1834bdd5c5761ab079fd04f463a89585df8fcae51a7bfe4229b1e02a652"
+SOURCE_DIR="hidapi-hidapi-$portVersion"
+# PATCHES="hidapi-$portVersion.patchset"
```

Comment:
Remove if unneeded

Reply:
Not yet, it is needed and needs to be worked on.